scalableminds/webknossos
# Refactor Backend: merge DataStoreSQL and DataStore

### URL of deployed dev instance (used for testing):
- https://refactorbackenddatastore.webknossos.xyz

### Steps to test:
- you should see datasets with thumbnails, be able to start explorational tracings

### Issues:
- contributes to #2521

------
